docs/edgeql/transactions.rst

```diff
@@ -47,16 +66,32 @@ Using an EdgeQL query string:
 .. code-block:: typescript
 
   client.transaction(async tx => {
-    await tx.execute(`insert Fish { name := 'Wanda' };`);
+    await tx.execute(`update BankCustomer 
+      filter .name = 'Customer1'
+      set { bank_balance := .bank_balance -10 }`);
+    await tx.execute(`update BankCustomer 
+      filter .name = 'Customer2'
+      set { bank_balance := .bank_balance +10 }`);
   });
 
 Using the querybuilder:
 
 .. code-block:: typescript
 
-  const query = e.insert(e.Fish, {
-    name: 'Wanda'
+  const query1 = e.update(e.BankCustomer, {
+    filter_single: {name: 'Customer1',
+    set: {
+      bank_balance -= 10
+    }
+  });
+
+  const query2 = e.update(e.BankCustomer, {
+    filter_single: {name: 'Customer2',
+    set: {
+      bank_balance += 10
+    }
   });
+
   client.transaction(async tx => {
     await query.run(tx);
   });
```

Comment:
Minor edits:

```suggestion
  const query1 = e.update(e.BankCustomer, {
    filter_single: { name: "Customer1" },
    set: {
      bank_balance: { "-=":  10 }
    },
  });

  const query2 = e.update(e.BankCustomer, {
    filter_single: { name: "Customer2" },
    set: {
      bank_balance: { "+=":  10 }
    },
  });

  client.transaction(async (tx) => {
    await query1.run(tx);
    await query2.run(tx);
  });
```

Reply:
Thanks! I had a feeling it was off by just about that amount.